haskell_mu_bench/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
would the result be the same if we omitted this command and went ahead with just `sh` instead of `bash`?

Reply:
Yeah, it would be the same. I can change it.

Reply:
Thank you, I like it minimal. :)